nettrine/phpstan.neon

```diff
@@ -0,0 +1,9 @@
+parameters:
+	level: max
+	paths:
+		- app
+
+	checkGenericClassInNonGenericObjectType: false
+
+	doctrine:
+		objectManagerLoader: tests/object-manager.php
```

Comment:
Misto to tests to hod do slozky /build/object-manager.php

Reply:
hotovo